docs/cadence/language/access-control.md

```diff
@@ -205,3 +219,193 @@ some.f[3] = 1
 // Valid: can call non-mutating methods on a public field in outer scope
 some.f.contains(0)
 ```
+
+## Entitlements
```

Comment:
IMO further extensions to entitlements can be separated out into their own section; so the mutability-changes, for example, can go into a separate section about mutability that references back to this section. I think however that since developers need to understand the most basic application of entitlements in order to write any kind of Cadence code with access control, we should have the introductory material on this page.